coq-builtin.elpi

```diff
@@ -792,7 +792,7 @@ external pred coq.env.primitive-projections i:inductive,
 
 % -- Universes --------------------------------------------------------
 
-% Univ.Universe.t
+% Sorts.t
 typeabbrev univ (ctype "univ").
```

Comment:
This is the thing I don't get, it can't be the right terminology

Reply:
I am fine with whatever you want to call that for the end-user. The only important point is that nobody should rely on the `Universe.t` type in a plugin without a good reason.